rimraf/index.js

```diff
@@ -1,11 +1,8 @@
-const rimraf = require('rimraf')
-const { promisify } = require('util')
-
-const rimrafP = promisify(rimraf)
+const {rimraf} = require('rimraf')
 
 module.exports = async (p) => {
   try {
-    await rimrafP(p)
+    await rimraf(p)
```

Comment:
It would be much better to use Node's built-in utility for this and avoid the dependency on `rimraf`:
```
rm(p, { recursive: true, force: true })
```